MapboxCoreNavigation/Feedback.swift

```diff
@@ -28,8 +28,8 @@ public enum FeedbackType: CustomStringConvertible {
         switch self {
         case .general:
             return "general"
-        case .incorrectVisual(_):
-            return "incorrect_visual"
+        case .looksIncorrect(_):
+            return "looks_incorrect"
```

Comment:
This needs to remain `incorrect_visual` for compatibility with the backend and consistency with the Android implementation. This string is never presented to users, so translation isn’t a concern.